NDB.Covid19/NDB.Covid19.Droid/Views/AuthenticationFlow/QuestionnairePageActivity.cs

```diff
@@ -261,9 +261,7 @@ void OnFail()
 
         void OnValidationFail()
```

Comment:
Is it called only when Date from MSIS is missing? Or can it be called due to other validations on this page?

Reply:
If MSIS date is missing, it will trigger a separate error aborting the flow. This is only invoked when the user has specified that they want to submit a date, but not specified one, e.g. dismissing the DatePickerDialog.